TramsDataApi/ResponseModels/CaseActions/NTI/NoticeToImprove/NoticeToImproveResponse.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+using System.Collections.Generic;
+using TramsDataApi.DatabaseModels;
+
+namespace TramsDataApi.ResponseModels.CaseActions.NTI.NoticeToImprove
+{
+    public class NoticeToImproveResponse
```

Comment:
Arguably there would be a different response depending on whether you're updating or creating, but that's just a comment, not a blocker.